recipe/meta.yaml

```diff
@@ -109,9 +119,9 @@ outputs:
         - {{ pin_subpackage("cuda-crt-dev_" + target_platform, exact=True) }}
         - {{ pin_subpackage("cuda-nvvm-dev_" + target_platform, exact=True) }}
         - libgcc {{ gcc_min_constraint }}  # [linux]
+        - arm-variant * {{ arm_variant_type }}    # [aarch64]
```

Comment:
Open question on this PR:

Should arm-variant be moved from run_constrained to run for the compiler packages? What about for packages in general?

For compiler packages, I argue *yes* because we cannot allow mixed-variant packages in the build environment because the compiler toolchain is divided into multiple packages which must match.

At runtime, I'm unsure, but I'm also unclear about what the value of allowing mixed arm variants is. Is it to reduce work for the conda solver by reducing dependencies by one package?